scripts/lib-franklin.js

```diff
@@ -535,7 +535,7 @@ export function decorateButtons(element) {
       const up = a.parentElement;
       const twoup = a.parentElement.parentElement;
       if (!a.querySelector('img')) {
-        if (up.childNodes.length === 1 && (up.tagName === 'P' || up.tagName === 'DIV')) {
+        if (up.childNodes.length === 1 && (up.tagName === 'DIV')) {
```

Comment:
Is there another way to achieve this? It would make it hard to upgrade the `lib-franklin.js` in the future...